### DISCUSSION TITLE
Webpage just says "success"

### DISCUSSION THREAD

**Discussion by AtticusPierce** [roleoroleo/sonoff-hack#124] [Q&A]
I'm trying to use this to connect to a camera and set it up on my network, but putting "http://ip-cam" in webpage doesn't reach the cam and putting "192.168.66.1" just gives a plaintext of "Success" no matter what else I put after it. I can't connect the camera through the app because it just endlessly plays that godsforsaken scream and the camera ignores it. I think it may be because the password on my network is too long, based on what some people have said on bug reports for the camera/app.

**Comment by roleoroleo** [comment]
Please, explain me better.
Are you able to detect the ip assigned to the cam (using the web interface of your router)?
If yes, try to connect to the cam with the browser with this ip in http (no https).

**Reply by AtticusPierce** [reply]
I don’t have it connected to my router, I’m using the built in SSID that it makes and connecting my computer to that. It has the local IP of 192.168.66.1 for that and when I go to “http://192.168.66.1” it just has “Success” as the title and content of the page.

**Reply by roleoroleo** [reply]
I didn't test this network mode and I don't know if the start procedure changes in this scenario.
Do ssh and telnet work?

**Reply by AtticusPierce** [reply]
Yes, I can get in with SSH, I assume telnet works but I didn’t try it because SSH is better. I think it would be a good idea to try loading NMCLI on there, I can see about trying that next time I have some free time from work…

**Reply by roleoroleo** [reply]
Tis system doesn't contain Network Manager.
Please check the running processes and the mounted file system.
```
# ps
# mount
```

**Reply by AtticusPierce** [reply]
Yes, I know. It is loading the hack just fine, but the web interface doesn't run. What is the name of the process for that?

Also, I can't use the app to connect to the wifi unless there's a way other than the sound.

**Reply by AtticusPierce** [reply]
Okay, I just borrowed an ethernet cable, the webserver DOES work when connected over ethernet, I wish I could do all this without having to hardwire, but at least this will work to install each one when I can get them to actually connect to the WiFi I set!

**Reply by roleoroleo** [reply]
If you connect the cam as a client you shouldn't have any problems.
